packages/lesswrong/server/resolvers/recentDiscussionFeed.ts

```diff
@@ -12,7 +13,7 @@ defineFeedResolver<Date>({
   resultTypesGraphQL: `
     postCommented: Post
     tagDiscussed: Tag
-    tagSubforumCommented: Tag
```

Comment:
I got some errors testing this locally due to (I assume) the cached version of the page still expecting `tagSubforumCommented`. Could you support both for a week or so, or just leave the name the same?

Reply:
Were any of the errors user-facing? I could only get errors to appear in the console, in which case I would prefer to keep the new name and just let the user refresh the page to fix it. I know you're OOO so I'll go ahead and change this back to the old name for now.

Reply:
Actually renaming it gives me an error around mismatching types anyway, so I'm going to leave this as-is.